scim/Dependencies.toml

```diff
@@ -0,0 +1,288 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
+
+# This file is auto-generated by Ballerina for managing dependency versions.
```

Comment:
Does this get auto-generated everytime we build the module? if not, shall we remove these comments about auto generation.

Reply:
Yes it get auto-generated even when it's removed